test/netpol-conformance/adminnetworkpolicy_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package netpol_conformance
+
+import (
+	"flag"
+	"fmt"
+	"testing"
+	"time"
+
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2econfig "k8s.io/kubernetes/test/e2e/framework/config"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	netpolv1alpha1 "sigs.k8s.io/network-policy-api/apis/v1alpha1"
+	"sigs.k8s.io/network-policy-api/conformance/tests"
+	netpolv1config "sigs.k8s.io/network-policy-api/conformance/utils/config"
+	"sigs.k8s.io/network-policy-api/conformance/utils/suite"
+)
+
+const (
+	showDebug                  = true
+	shouldCleanup              = true
+	enableAllSupportedFeatures = true
+	NetworkPolicyAPIRepoURL    = "https://raw.githubusercontent.com/kubernetes-sigs/network-policy-api/v0.1.1"
```

Comment:
Seems this is not consistent with the version `v0.1.0` you used in `test-netpol-v2-conformance-kind.sh`

Reply:
Yes. For the installation manifest, only available version is `v0.1.0` which is the major release. `v0.1.1` however contains some changes to the conformance test files that we should use for testing